charts/perses/templates/ingress.yaml

```diff
@@ -1,18 +1,7 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "perses.fullname" . -}}
 {{- $svcPort := .Values.service.port -}}
-{{- if and .Values.ingress.ingressClassName (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
```

Comment:
the list of reasons is as follows:

1. k8s 1.18 has reached the end-of-life more than 3 years ago. I'm pretty sure everyone had enough time to update to a newer version
2. `kubernetes.io/ingress.class` has been deprecated since 1.18 https://kubernetes.io/docs/concepts/services-networking/ingress/#deprecated-annotation
3. There is code below that renders `.Values.ingress.annotations` at line 21
4. Removing these version checks declutters already difficult to read manifest templates.

1 and 4 applies to all version checks I removed in this PR

Reply:
Makes sense and it was a solid answer, thanks for that.

Reply:
yeay! thanks!